### DISCUSSION TITLE
Miracle: Shield of Faith

### DISCUSSION THREAD

**Discussion by Hoppelite** [Lambosaurus/hives-and-torches#52] [Divine Action]
The target gains a 1d4 bonus to their defence for the rest of combat.

**Comment by Hoppelite** [comment]
I was thinking maybe this should just be a base increase.

**Reply by Lambosaurus** [reply]
I feel like +2 ac is the way to go. Defence is pretty strong, and I think the deviation is pretty dangerous here. 1 is a letdown, and 4 is super fuckin strong.

But it might be a fun kind of random too?
